apps/sms/views/conversation/js/conversation.js

```diff
@@ -688,7 +688,7 @@ var ConversationView = {
       // Populate draft if there is one
       // TODO merge with handleDraft ? Bug 1164431
       Drafts.request().then(() => {
-        this.draft = Threads.get(threadId).getDraft();
+        this.draft = Threads.get(threadId) && Threads.get(threadId).getDraft();
```

Comment:
@azasypkin  umm, it was throwing type error, i.e it's null, for some unit test (not related to this patch, this issue was before only). So, thought to fix it.